packages/renderer/src/lib/ContainerList.svelte

```diff
@@ -320,7 +322,7 @@ function toggleAllContainerGroups(value: boolean) {
       {#each containerGroups as containerGroup}
         <tbody>
           {#if containerGroup.type === ContainerGroupInfoTypeUI.COMPOSE || containerGroup.type === ContainerGroupInfoTypeUI.POD}
```

Comment:
Will it show pod actions on a compose object ?

Reply:
Ah, you're right. I've pushed a fix.